### PR TITLE
[wasm][debugger] Fix type argument

### PR DESCRIPTION
**Pull request by lewing**
fix numbers inspection in js

### CODE REVIEW

File: mono/mini/mini-wasm-debugger.c [hunk 1]
```diff
@@ -50,7 +50,7 @@ extern void mono_wasm_add_func_var (const char*, const char*, guint64);
 extern void mono_wasm_add_properties_var (const char*, gint32);
 extern void mono_wasm_add_array_item (int);
 extern void mono_wasm_set_is_async_method (guint64);
-extern void mono_wasm_add_typed_value (const char *type, const char *str_value, int int_value);
+extern void mono_wasm_add_typed_value (const char *type, const char *str_value, int double);
```
Comment (lambdageek) [inline mono/mini/mini-wasm-debugger.c:53]: You just changed the parameter name, not it's type

Comment (lewing) [reply mono/mini/mini-wasm-debugger.c:53]: whoops
:)

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build Linux ARMv7 Interpreter

**Comment by lewing** [comment]
fixes https://github.com/mono/mono/issues/19702

**Comment by lewing** [comment]
@monojenkins backport 3.2-wasm

**Pull request merged**